.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Related issue
+
+Closes: #ISSUE_NUMBER
+
+
+## Description
+* Describe the changes in this pull request
+
+## Additional Testing
+Describe any testing done in addition to existing unit tests
+
+## Additional context
+Please provide any additional information related to this PR
+
+---
+*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*
```

Comment:
Check template for porting-assistant-client. They have to be consistent. We can either take their template or update it.

Reply:
We updated the templates on the PA side so now the templates will be in sync: https://github.com/aws/porting-assistant-dotnet-client/pull/63